WickedEngine/wiShaderCompiler.cpp

```diff
@@ -443,139 +438,6 @@ namespace wiShaderCompiler
 	}
 #endif // SHADERCOMPILER_ENABLED_DXCOMPILER
 
-#ifdef SHADERCOMPILER_ENABLED_D3DCOMPILER
```

Comment:
I actually wanted to keep this here, to have a reference of how the old d3dcompiler interface can be used. This will come in handy when somebody searches for d3d compile functions on github. Keeping it here and disabled is fine.